quickstarts/Models.ipynb

```diff
@@ -175,7 +175,7 @@
       "outputs": [],
```

Comment:
<p>Shouldn't this be genai.get_model('models/gemini-1.5-flash-latest')</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>